app/coffee/Features/DocumentUpdater/DocumentUpdaterHandler.coffee

```diff
@@ -153,6 +153,22 @@ module.exports = DocumentUpdaterHandler =
 				logger.error project_id:project_id, url: url, "doc updater returned a non-success status code: #{res.statusCode}"
 				callback new Error("doc updater returned a non-success status code: #{res.statusCode}")
 
+	clearProjectState: (project_id, callback = (error) ->) ->
+		timer = new metrics.Timer("clear-project-state")
+		url = "#{settings.apis.documentupdater.url}/project/#{project_id}/clear"
```

Comment:
Let's make this `#{settings.apis.documentupdater.url}/project/#{project_id}/clearState`, to avoid confusion with clearing everything out of the doc updater as on a delete.

Reply:
(I don't need to reapprove the corresponding change in doc updater)